src/analysis/retail/evoker/augmentation/modules/Styling.scss

```
@@ -78,3 +78,34 @@
 .npc {
   color: #ff6a6a;
 }
+
+.table-container {
+  margin-bottom: 1rem; // Adds space below the table
+}
+
+.button-container {
+  display: flex;
+  justify-content: space-between; // This spreads out the buttons to the width of the container
+  align-items: center; // This aligns buttons in the middle vertically if they have different heights
+  padding: 0 10px; // Add padding to the left and right
+  margin-top: 1rem; // Adds space above the button container
```

If this is removed and you put in `gap: 20px` instead  it would line the buttons up nicely.

---

I think this is the only one I am stuck on :)  removing the `table-container` makes sense, but after some trial and error I couldn't get it to look right without leaving the button container and keep the following style:
 `.button-container {
  display: flex;
  justify-content: space-evenly; 
  margin-top: 2rem;
}`

---

```suggestion
 gap: 20px
```
Is what I mean, you still keep the container.